src/en/charms-scaling.md

```diff
@@ -1,88 +1,68 @@
 Title: Scaling applications
-TODO:  The Scaling Back section should just reference charms-destroy.html (remove-unit)
-       Critical: review required
+TODO:  More direction ito how users discover "built-in scaling" (e.g. the store is not clear on wordpress)
 
 # Scaling applications
 
-One of the killer features of computing in the cloud is that it (should)
-seamlessly allow you to scale up or down your applications to meet your needs
-and whims. Juju not only makes it simple to deploy applications, but crucially
-makes it easy to manage them too. It won't anticipate you getting slashdotted
-or on the front page of hacker news (yet), but it does mean that when you do
-you can reliably scale your applications to meet the demand.
+The capability of a service to adjust its resource footprint to a level
+appropriate for fulfilling client demands placed upon it is known as
+*scalability*. Scaling *vertically* affects the resources of existing machines
+(memory, CPU, disk space) whereas scaling *horizontally* involves the number of
+units available.
 
+Scaling with Juju amounts to horizontal scaling where "scaling up" adds units
```

Comment:
I think this is a little unclear after a sentence where unit scaling was an example of horizontal, although I know what you mean. Maybe something more like 'Juju abstracts horizontal scaling, where adding and removing units becomes scaling up and down within the Juju environment', or something much better.

Reply:
This sentence was more of a reinforcement of the previous sentence so I just removed it.